modules/blocks/utils.js

```diff
@@ -66,8 +66,8 @@ function Utils(
  * Normalize blocks and their transactions
  *
  * @method readDbRows
- * @param  {[Object]} rows Data from full_blocks_list view
- * @return {[Object]} blocks Normalized list of blocks with transactions
+ * @param  {Array} rows Data from full_blocks_list view
+ * @return {Array} blocks Normalized list of blocks with transactions
```

Comment:
Is this related to this PR?

Reply:
oops, included it by accident. 

It fixes a bug, because it resulted in an error when trying to build the JSDoc HTML files.
Can I leave it here or should it be part of the PR for the modules-folder (coming soon)?